client/components/search/index.jsx

```diff
@@ -38,7 +38,8 @@ module.exports = React.createClass( {
 		onKeyDown: React.PropTypes.func,
 		disableAutocorrect: React.PropTypes.bool,
 		onBlur: React.PropTypes.func,
-		searching: React.PropTypes.bool
+		searching: React.PropTypes.bool,
+		forceCloseButton: React.PropTypes.bool
```

Comment:
> so it has a function.

It very much has a function here. It closes the add new site input. There seems to be some confusion. We're not talking about the search input that appears above the list of sites; We're talking about the input (which is using the search component) that appears when you click the add button. This add input (which again, uses the search component) transition in place and activates an "add mode." The only way to exit "add mode" is with the add inputs `x` icon — which only shows up once you type. So, the problem is that there's no way to exit "add mode" without that `x` icon.